src/ext/issuance.js

```diff
@@ -41,6 +46,25 @@ function signReqCF(url) {
     return xhrInfo;
 }
 
+function signReqHC(url) {
+    let reqUrl = url.href;
+    const isIssuerUrl = ACTIVE_CONFIG["issue-action"]["urls"]
```

Comment:
As above, I would personally define `ACTIVE_CONFIG["issue-action"]["urls"]` in background.js and export it here.